Dockerfile

```diff
@@ -20,12 +20,16 @@ RUN apt-get -y install libzmq3-dev
 
 # switch to the freshly cloned verge project & compile it
 WORKDIR /app/verge
+RUN git checkout master
```

Comment:
Can we use the latest known tag here? 

```bash
# Get new tags from remote
git fetch --tags

# Get latest tag name
latestTag=$(git describe --tags `git rev-list --tags --max-count=1`)

# Checkout latest tag
git checkout $latestTag
```

Reply:
Then you never have the latest. And you would want to lock the tag for specific versions